static-analysis/autodispose-lint/src/test/kotlin/com/uber/autodispose/lint/AutoDisposeDetectorTest.kt

```diff
@@ -621,4 +621,36 @@ class AutoDisposeDetectorTest {
         .run()
         .expectClean()
   }
+
+  @Test fun subscribeWithCapturedDisposable() {
```

Comment:
We'll need failure cases here too. Can you try to reuse some of the tests we had for the error prone check?

Reply:
We already have a failing `subscribeWith` here https://github.com/uber/AutoDispose/blob/986b4b206b75926bcf2ba514f5bccb0ef9ec504d/static-analysis/autodispose-lint/src/test/kotlin/com/uber/autodispose/lint/AutoDisposeDetectorTest.kt#L117

Reply:
I meant capturing a returned type that doesn't implement `Disposable`